### ISSUE TITLE
Extend to 2 000 000 000 please

### ISSUE THREAD

**Issue by bondarenkod** [AceLewis/my_first_calculator.py#16]
Thank you for very useful calculator. Could you extend range of this calculator? -2147000000000...2147000000000 looks like good.
Thank you!

**Comment by aramann** [comment]
My python crashes on 0...1000, and you want to do it -2147000000000...2147000000000?????

**Comment by AlexTsygankov** [comment]
Yeah, i think we realy need this feature. It would be awesome.

**Comment by SDKiller** [comment]
LOL

**Comment by rudrathegreat** [comment]
true... That would be good. Maybe to -1000000000000000000000000 to 12000000000000000000000000
